packages/fast-tooling/src/message-system-service/shortcuts-service-actions/shortcuts.service-action.redo.ts

```diff
@@ -0,0 +1,36 @@
+import { ShortcutsAction } from "../shortcuts.service-action";
+import { MessageSystem, MessageSystemType } from "../../message-system";
+import { shortcutsId } from "../shortcuts.service";
+import { MessageSystemHistoryTypeAction } from "../../message-system/message-system.utilities.props";
+
+/**
+ * @alpha
+ */
+export default function ShortcutsActionRedo(
+    messageSystem: MessageSystem
+): ShortcutsAction {
+    return new ShortcutsAction({
+        id: "redo",
+        name: "Redo",
+        keys: [
```

Comment:
Why not "ctrl+y"? That seems to be the standard for redo in Msft products (VS Code, Word, etc).

Reply:
Hmmm, that's a good question, I'll toss it @bheston for his opinion on what designers might expect for "undo" shortcut in a design tool.

Reply:
I've gotten feedback that both should be implemented but that Ctrl + Shift + Z is preferred, so we'll go with that for now and see how we can use multiple key combinations for a single action in future.